docs/evaluation/how_to_guides/evaluation/evaluate_with_attachments.mdx

```diff
@@ -0,0 +1,103 @@
+import {
+  CodeTabs,
+  python,
+  typescript,
+  PythonBlock,
+  TypeScriptBlock,
+} from "@site/src/components/InstructionsWithCode";
+
+# Evaluate an LLM application with attachments
```

Comment:
feels like this describes the implementation not the problem. maybe `Evaluate applications on large files` or `Evaluate applications with large file inputs` or something like that